breakpad-symbols/src/sym_file/types.rs

```diff
@@ -222,3 +239,27 @@ pub struct SymbolFile {
     /// still be correct.
     pub cfi_eval_corruptions: u64,
 }
+
+impl SymbolFile {
+    /// Returns the closest function to the address for which `function.address <= address`.
+    pub fn find_nearest_function(&self, address: u64) -> Option<&Function> {
```

Comment:
Feels like this wants a better name. `find_nearest_function_before`?

Reply:
Sure, sounds good.